src/modules/cluster/cluster-detail/cluster-detail.js

```diff
@@ -9,19 +9,28 @@
     function clusterDetailController($state, $stateParams, utils) {
 
         var vm = this;
+        vm.tabList = {};
+        vm.tabName = "";
         vm.setTab = setTab;
         vm.isTabSet = isTabSet;
 
         vm.clusterId = $stateParams.clusterId;
-        vm.tabList = {
-            Volumes: 1
-        };
-        vm.activeTab = vm.tabList.Volumes;
 
         utils.getObjectList("Cluster").then(function(list) {
             vm.cluster = list.filter(function(cluster) {
                 return cluster.cluster_id === vm.clusterId;
             })[0];
+
+            /* Setting up the tab based on ceph/gluster */
+            if(vm.cluster.sds_name === "gluster") {
```

Comment:
@anivargi In APi response there is only one properties called "sds_name" , by using this only we can get to know which kind of cluster(ceph, gluster) it is ,  and again if cluster is "ceph" specific than instead of "volume" we should have "pool" tab. and same for "gluster" as well . if cluster is "gluster" specific than instead of "pool" we should have "volume" tab.

I mean "tabName" is dependent on sds_name.. so here just i am adding tab for appropriate cluster